tools/travis/build.sh

```diff
@@ -38,8 +38,16 @@ TERM=dumb ./gradlew install
 echo "openwhisk.home=$WHISKDIR" > whisk.properties
 echo "vcap.services.file=" >> whisk.properties
 
-# Build runtime
+# Build runtime and dependencies
 cd $ROOTDIR
+
+# Build runtime
 TERM=dumb ./gradlew \
 :core:nodejs6Action:distDocker \
 :core:nodejs8Action:distDocker
+
+# Build test dependency containers
+docker build --tag nodejs6docker ./tests/dat/docker/nodejs6docker/
```

Comment:
Don't build the images manually use a gradle project and the `distDocker` task

Reply:
you can create two project and include them, then just run `./gradlew distDocker` it will build all 4 images